src/randoop/main/GenInputsAbstract.java

```diff
@@ -366,36 +318,42 @@ public GenInputsAbstract(String command, String pitch,
    */
   @Option("Which exceptional tests to include in the regression test suite")
   public static ExceptionAssertionsMode regression_assertions_about_exceptions = ExceptionAssertionsMode.ALL;
-
-
-  // TODO make an enum. (But presently Options package requires upper-case
-  // strings for enums, which will require changes to Make targets, plugin, etc.)
-  /** For details, see the Javadoc documentation for {@link randoop.DefaultTestFilter}. */
-  @Option("What kinds of tests to output: pass, fail, or all")
-  public static String output_tests = "all";
   
-  public final static String ALL = "all";
-  public final static String FAIL = "fail";
-  public final static String PASS = "pass";
-
-  @Option("Simplify (shorten) failed tests while preserving failure behavior")
-  public static boolean simplify_failed_tests = false;
-
+  /**
+   * The possible values for exception allocation command-line arguments.
+   */
+  public static enum ExceptionAllocation {
+    /** Occurrence of exception should be considered an error */
+    ERROR,
+    /** Occurrence of exception should be considered invalid */
+    INVALID,
+    /** Occurrence of exception should be considered expected behavior */
+    REGRESSION
```

Comment:
Given the description just above, how about renaming this as EXPECTED?  That focuses more on the behavior rather than Randoop's mechanics of outputting tests.